docker/typescript/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM base-sdk:latest
+MAINTAINER Rodney Gomes <rodney.gomes@looker.com>
```

Comment:
that should be your name @muckweaver

Reply:
I was hoping you'd take the blame on this.